DirectProgramming/C++SYCL_FPGA/Tutorials/Features/max_interleaving/CMakeLists.txt

```diff
@@ -39,7 +39,7 @@ endif()
 set(USER_FPGA_FLAGS ${USER_FPGA_FLAGS})
 
 # Use cmake -DUSER_FLAGS=<flags> to set extra flags for general compilation.
-set(USER_FLAGS ${USER_FLAGS})
+set(USER_FLAGS -Wno-unused-label;${USER_FLAGS})
```

Comment:
```suggestion
# This design uses labels to improve report readability, so add -Wno-unused-label to disable warnings about unused labels. 
set(USER_FLAGS -Wno-unused-label;${USER_FLAGS})
```

Reply:
probably should explain this for people who wonder why there's a random flag in there

Reply:
Updated.